src/InteractiveWindow/VisualStudio/InteractiveWindow.vsct

```diff
@@ -231,6 +231,17 @@
             -->
 
       <!-- Text editor commands -->
+      <!-- TODO: Remove once https://github.com/dotnet/roslyn/issues/8927 is done. -->
+      <Button guid="guidInteractiveWindowCmdSet" id="cmdidExecuteInInteractiveWindow" priority="0x101" type="Button">
+        <Parent guid="guidInteractiveWindowCmdSet" id="TextEditorGroup" />
+        <CommandFlag>DynamicVisibility</CommandFlag>
+        <CommandFlag>DefaultInvisible</CommandFlag>
+        <Strings>
+          <ButtonText>Execute in Interactive</ButtonText>
```

Comment:
This menu button was already defined in the exact form in previous releases but was removed in an effort to merge the F# and C# execute in interactive command (see #8161).